llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1572,7 +1572,8 @@ def FeatureISAVersion12 : FeatureSet<
    FeatureVGPRSingleUseHintInsts,
    FeatureScalarDwordx3Loads,
    FeatureDPPSrc1SGPR,
-   FeatureMaxHardClauseLength32]>;
+   FeatureMaxHardClauseLength32,
+   Feature1_5xVGPRs]>;
```

Comment:
Is it true for all GFX12? The SGP seems to say something else.

Reply:
Yes it is true for gfx1200 and gfx1201.